src/components/Question.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+const Question = ({ question, answer }) => {
+  const [viewAnswer, setViewAnswer] = useState(false);
+
+  function arrowClickHandler() {
+    setViewAnswer(!viewAnswer);
```

Comment:
when using value from previous state to produce new one, use callback.
`setViewAnswer(answer => !answer)`
This will guarantee having latest state value when using it